dashboard/src/components/InfoCard/InfoCard.scss

```diff
@@ -32,3 +32,11 @@
 .ListItem__content__info_tag-2 {
   background-color: #1598cb;
 }
+
+.ListItem__content__info_tag.stable {
```

Comment:
The issue was that it's also being imported by the ServiceCatalogList components, but maybe since they will eventually be combined we can keep it as it was.